Set-1/Solved Programs/KattapaReadyForBattle/KattapaReadyForBattle.java

```diff
@@ -0,0 +1,41 @@
+package com.problems;
+
+import java.util.logging.Logger;
+
+public class KattapaReadyForBattle {
+
+    public static String isReadyForBattle(int n,int[] soldierWeapons){
```

Comment:
why static method

Reply:
static method invoked without the need for creating an instance of a class